File: backend/src/main/java/com/portfolio/backend/Model/Habilidad.java
package com.portfolio.backend.Model;

import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class Habilidad {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long idHabilidad;
    @Basic
    private String nombreHabilidad;
    private Integer puntuacion;
    private String descripcion;

    public Habilidad() {
    }

    public Habilidad(Long idHabilidad, String nombreHabilidad, Integer puntuacion, String descripcion) {
        this.idHabilidad = idHabilidad;
        this.nombreHabilidad = nombreHabilidad;
        this.puntuacion = puntuacion;
        this.descripcion = descripcion;
    }
}

File: FrontEnd/src/app/entidades/habilidad.ts
export class Habilidad {
    nombreHabilidad:string;
    puntuacion:number;
    descripcion: string;

    constructor (nombreHabilidad:string, puntuacion:number, descripcion:string){
        this.nombreHabilidad=nombreHabilidad;
        this.puntuacion=puntuacion;
        this.descripcion=descripcion
    }
}
